apps/condo/domains/miniapp/schema/B2BAppContext.js

```diff
@@ -135,6 +135,13 @@ const B2BAppContext = new GQLListSchema('B2BAppContext', {
 
             if (isSoftDeletedHappened) {
                 await deleteB2BAppRoles.delay(appId, organizationId)
+                const accessTokens = await find('B2BAccessToken', {
+                    context: { id: updatedItem.id },
+                    deletedAt: null,
+                })
+                for (const accessToken of accessTokens) {
+                    await B2BAccessToken.update(context, accessToken.id, { dv: accessToken.dv, sender, deletedAt: new Date().toISOString() }, 'id')
```

Comment:
If app failed here, should there be some cron task to kill tokens, it'll be more secure

Reply:
For start there will be only few tokens, so task can appear later